app/controllers/admin/merchants_controller.rb

```diff
@@ -8,8 +8,11 @@ def update
     merchant.enable_disable
     if merchant.disabled
       merchant.deactivate_items
+      flash[:notice] = "Merchant has been disabled"
+    else
+      merchant.activate_items
+      flash[:notice] = "Merchant has been enabled"
     end
```

Comment:
Add conditional for enabling and disabling a merchant as well as activating and deactivating the corresponding items